numpyro/mcmc.py

```diff
@@ -52,6 +53,12 @@ def _euclidean_ke(inverse_mass_matrix, r):
     return 0.5 * np.dot(v, r)
 
 
+def diagnostics_str(hmc_state):
+    return '{} steps of size {:.2f}. E[p(acc.)]={:.2f}'.format(hmc_state.num_steps,
```

Comment:
opss, `E[p(acc.)]` would be tricky to guess what it means. I think we can just use `acc. rate` as in Pyro. We can add docs to this function to make it clearer for users.

nit: `{:.2f}.` might be not a good format for step_size (because its value is usually smaller than that). IMO, as in Pyro, "{:.2e};" is a better format

Reply:
It's not really acceptance rate though which is typically higher and what we report in Pyro (also Stan I guess). Maybe mean acc. probability or something. tbh - this was just to save space. :D